src/dcm230/test_dcm230.py

```diff
@@ -42,21 +42,124 @@ def test_read_register() -> None:
         _ = meter._read_registers(address, count, reg_type)
 
 
-def test_V() -> None:
-    """Test get v."""
+def test_range_validation() -> None:
+    """Test range validation."""
+    client = MagicMock()
+    meter = Dcm230(1, client)
+
+    """Test 1: Should not pass due to out of range."""
+    for name, spec in Dcm230._register_specs.items():  # type: ignore[attr-defined]
+        if not spec.range:
+            continue
+
+        client.write_registers.return_value.isError.return_value = False
+        with nullcontext():
+            _ = setattr(meter, name, spec.min)
```

Comment:
What I mean is to test for out of range on the minimum side as well, so we would expect a ValueError here.

Reply:
I want to test under the minimum.